reana_cluster/cli/__init__.py

```diff
@@ -73,7 +73,7 @@ def cli(ctx, loglevel, skip_validation, file):
     logging.basicConfig(
         format=DEBUG_LOG_FORMAT if loglevel == 'debug' else LOG_FORMAT,
         stream=sys.stderr,
-        level=logging.DEBUG if loglevel == 'debug' else logging.INFO)
+        level=logging.DEBUG if loglevel == 'debug' else logging.ERROR)
```

Comment:
Hmm, maybe `WARNING` would be better as a default level, not to miss any important stuff...